nemo_skills/evaluation/evaluator/math.py

```diff
@@ -53,6 +53,8 @@ class LeanEvaluatorConfig:
     timeout: float = 30.0
     final_answer_key: str = "**FINAL ANSWER**"
     restate_formal_statement: bool = True
+    # Which code block to extract when multiple are present: "first" or "last"
+    code_block_from: str = "last"
```

Comment:
maybe also rename this one for consistency? Maybe we use the same name for this argument in all places, I guess something like extract_code_mode. Not sure what's the best name, but keeping it consistent seems like a good idea